definitions/npm/bluebird_v3.x.x/flow_v0.32.x-/bluebird_v3.x.x.js

```diff
@@ -142,6 +142,7 @@ declare class Bluebird$Promise<R> {
   each<T, U>(iterator: (item: T, index: number, arrayLength: number) => Bluebird$Promisable<U>): Bluebird$Promise<Array<T>>;
   asCallback<T>(callback: (error: ?any, value?: T) => any, options?: Bluebird$SpreadOption): void;
   return<T>(value: T): Bluebird$Promise<T>;
+  spread(...args: Array<any>): Bluebird$Promise<any>;
```

Comment:
Hmm, not sure how `flow` will treat different promise results in one array as a generic... do you mind adding a simple test in your test file :-)? If this works it would be better to use generics than `any`